synapse/axon.py

```diff
@@ -921,6 +921,25 @@ async def wget(self, url, params=None, headers=None, json=None, body=None, metho
         '''
         logger.debug(f'Wget called for [{url}].', extra=await self.getLogExtra(url=url))
 
+        if headers:
+            headers = dict(headers)
+        if headers is None:
+            headers = {}
+
+        # lower the keys
+        lheaders = dict((h.lower(), v) for h, v in headers.items())
+
+        defheaders = {
```

Comment:
how would we exclude one of these defaults ( to make it not present rather than have a new value? )